src/internal/xml-parser.ts

```diff
@@ -4,6 +4,7 @@ import { XMLParser } from 'fast-xml-parser'
 
 import * as errors from '../errors.ts'
 import type { BucketItemWithMetadata } from '../minio'
```

Comment:
Also this one

Reply:
:+1:  Yes. i have updated the type reference.